iib/workers/tasks/build.py

```diff
@@ -840,6 +908,39 @@ def handle_add_request(
             (prebuild_info['distribution_scope'] in ['dev', 'stage']),
         )
 
+        # Add the max ocp version property
+        # We need to ensure that any bundle which has deprecated/removed API(s) in 1.22/ocp 4.9
+        # will have this property to prevent users from upgrading clusters to 4.9 before upgrading
+        # the operator installed to a version that is compatible with 4.9
+
+        # Get the CSV name and version (not just the bundle path)
+        db_path = temp_dir + "/database/index.db"
+        port, rpc_proc = _serve_index_registry(db_path)
+
+        rawBundles = run_cmd(
+            ['grpcurl', '-plaintext', f'localhost:{port}', 'api.Registry/ListBundles'],
```

Comment:
I wonder if we should mention this dependency somewhere

Reply:
on grpcurl?

Reply:
yeah

Reply:
@midnightercz Just FYI: We already depend on that - this was not newly introduced.
I do agree with you that we should have it mentioned somewhere. But I think we should do it for every non python dependency that we have (opm, podman, buildah, grpcurl, ...) - not part of this PR